dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -283,4 +283,19 @@
             </column>
         </addColumn>
     </changeSet>
+    <changeSet author="hyunnnaye" id="removeAuthorAndEmailInEntry">
+        <sql dbms="postgresql">
+            /* delete author and email column in an entry */
```

Comment:
I believe this is OK, but just to confirm, no data will be lost this way, right? The previous data was copied, and no new values have been put in these columns?

Reply:
I believe the entry author and email were originally set using the `VersionMetadata` author and email since it's [synced from the default version](https://github.com/dockstore/dockstore/blob/8dc701e48b6ee401b7ef8fc069e20fb723a18cff/dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java#L465-L470). The previous data was copied in https://github.com/dockstore/dockstore/pull/4200 when the authors and emails were migrated from `VersionMetadata` to `Authors`, so it should be okay to drop the entry columns.

I don't think there are any new values in the columns because the `Entry.setAuthor()` function isn't used anywhere